core/project.go

```diff
@@ -59,7 +67,32 @@ func (p *Project) Deploy(environment string, output io.Writer, force bool) []err
 
 	file := NewDockerfile(blob, p, r, e)
 
-	return d.Deploy(p, r, file, output, force)
+	errs = d.Deploy(p, r, file, output, force)
+	if len(errs) == 0 {
+		p.afterSuccessfulDeploy(status)
+	}
+	return errs
+}
+
+func (p *Project) afterSuccessfulDeploy(status *ProjectStatus) {
```

Comment:
Why just after success, we can make this generic and in the payload include if was success or now and the errors